conf_files/testing.yaml

```diff
@@ -96,10 +96,11 @@ flat_fields:
     - deux
     - drei
     - quattro
-  required_exposures: 1  # Per filter per camera
-  max_attempts: 1  # Per filter per camera
-  max_exptime: 1 s
-  min_exptime: 0.0001 s
+  autoflats:
+    required_exposures: 2  # Per filter per camera
+    max_exposures: 2  # Per filter per camera
+    max_exptime: 1 s
+    min_exptime: 0.0001 s
```

Comment:
Maybe not relevant for this PR, but given the minimum exposure time for flats is 0.0001s, is it an easy config thing to change the sun altitude at which flats start, so that they can begin maybe 1 hour - 30 mins before sunset?

Reply:
yes, the flat field horizon is already configurable!

Reply:
I *think* we might eventually do both "dayflats" and "twilight flats" everyday (and same thing in the morning?). That'll increase the likelihood of us obtaining some sort of flats on a particular day and will eventually give us a broader spectral range to fine-tune the flats for a particular science case. What we'd want to do is monitor how the spectrum of the sky during a clear day changes from well before sunset into astronomical twilight. It'll start blue and become redder as the sunset progresses. So eventually I think we'll have `min_daytime_extime` etc